org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaCompletionExtensionsTest.java

```diff
@@ -301,6 +301,33 @@ public void completionWithoutStartBracket() throws BadLocationException {
 		
 	}
 
+	@Test

```

Comment:
Please add test with xml which not define xmlns="http://www.w3.org/2001/XMLSchema-instance" to check that xml:| doesn't provide xsi completions